Tests/Fixtures/Generated/JSON/inline-swift4/all-customName.swift

```diff
@@ -10,12 +10,6 @@ import Foundation
 
 // swiftlint:disable identifier_name line_length number_separator type_body_length
 internal enum CustomJSON {
-  internal enum Configuration {
-    internal static let apiVersion: String = "2"
-    internal static let country: Any? = nil
-    internal static let environment: String = "staging"
-    internal static let options: [String: Any] = ["screen-order": ["1", "2", "3"]]
-  }
   internal enum Documents {
```

Comment:
I see that this Generated output is containing values from the YAML contexts: `enum Documents`, `enum GroceryList`, `enum Mapping`… those are the result of parsing `documents.yml`, `grocery-list.yml`, `mappings.yml` input files that are located in `Fixtures/StencilContexts/Yaml`

Instead for consistency's sake we should use the `JSON` input files and especially the `StencilContexts/Json/*` they generate as input for the `JSON` tests.

This discrepency there is likely due to `Tests/TemplatesTests/JsonTests.swift` which needs to be updated to change every `resourceDirectory:` parameter used in `test(…)` calls to be `.json` and not `.yaml` anymore